versioned_sidebars/version-3.9.0-sidebars.json

```diff
@@ -383,8 +383,7 @@
           "collapsed": false,
           "items": [
             "user-guide/add-cd-pipeline",
-            "user-guide/manage-environments",
-            "user-guide/add-quality-gate"
```

Comment:
why did you remove this?

Reply:
remove just "user-guide/add-quality-gate" and "," in 386 string